datasets/1.0/audio_test/metadata.json

```diff
@@ -42,7 +42,8 @@
     "separator": "cr:separator",
     "source": "cr:source",
     "subField": "cr:subField",
-    "transform": "cr:transform"
+    "transform": "cr:transform",
+    "wd": "https://www.wikidata.org/wiki/"
```

Comment:
wd shouldn't be in the default context. I'm of the opinion that it should be added only when needed (the context is long enough). We can discuss this.

Reply:
Sure. Done.